content/learn/courses/anchor-positioning.md

```diff
@@ -0,0 +1,278 @@
+### Fewer dropdown bugs
+
+What happens to dropdowns that are close to viewport edges,
+or as the user scrolls?
+CSS anchor positioning offers a robust feature set to handle
+these literal edge cases.
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+### Fast page performance
+
+JavaScript positioning libraries require a lot of render-blocking
+computations to do what they do.
+Moving that load to the browser reduces performance bottlenecks.
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+### Future-proof component libraries
+
+JavaScript-based solutions require updates and maintenance.
+CSS anchor positioning is part of the browser spec,
+and will still work in 10 years.
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+### Full support now
+
+Not all your users have a browser that supports CSS anchor positioning yet. I'll
+show you how to use OddBird's [anchor positioning
+polyfill](https://anchor-polyfill.netlify.app/) to support all your users.
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+### Advanced use cases
+
+The ability to declaratively reference the size and position
+of another element unlocks all sorts of creative possibilities.
+
+{% endcall %}
+
+{% endcall %}
+
+{% endcall %}
+
+## Anchor positioning demo
+
+{{ embed.codepen(
+  id='wBvEavV',
+  title='Anchor Positioning Demo',
+  user='jamessw',
+  height=300
+) }}
+
+## 9 weekly emails
+
+{% call layout.grid(
+  'narrow-columns',
+  attrs={'style': '--column: page'}
+) %}
+
+{% call layout.block('column') %}
+
+- Placement with `position-area`
+- Using a position with `anchor()`
+- Using a size with `anchor-size()`
+- Reusing anchor declarations
+- Positioning popovers
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+- Troubleshooting anchor positioning
+- Scroll visibility
+- Handling element overflow
+- Use it now with a polyfill
+
+{% endcall %}
+
+{% endcall %}
+
+{% call layout.block() %}
+
+{{ utility.main_action(
+  content='Sign up to get your first email',
+  url='#sign-up'
+) }}
+
+{% endcall %}
+
+<div style="margin-bottom: 1rem">
+
+## What this email course teaches
+
+You will learn about anchor positioning, scoping your anchors, handling
+overflow, anchoring with the Shadow DOM, positioning popovers, using Dev Tools
+to troubleshoot anchors, pseudo elements that can be anchors, and new
+possibilities for innovative layouts.
+
+</div>
+
+## About James
+
+{% set bird = collections.birds | authorPage('james') %}
+
+{{ birds.card(bird=bird, extended=extended) }}
+
+{{ utility.main_action(
+  content='Subscribe to the free course',
+  url='#sign-up'
+) }}
+
+{{ quotes.grid(press) }}
+
+## <a id="sign-up">Create performant layouts and resilient dropdowns</a>
+
+{% call layout.grid(
+  'narrow-columns',
+  attrs={'style': '--column: page'}
+) %}
+
+{% call layout.block('column') %}
+
+{{ embed.icon_block(
+  data=included,
+  grid='fit'
+) }}
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+### Sign up for email course
```

Comment:
Instead of having this heading outside of the form, should we change the oddnews_signup macro to allow alternative headings/subheadings? I think then the heading would appear to be more associated with the form.

Reply:
I do think it would be nice to switch to the oddnews_signup style here for the reasons you list, @dvdherron. 

**Header**: Email Course Sign Up
**Subheader**: Be among the first to discover new layout possibilities with anchor positioning.

Reply:
Ok, this should be fairly easy. I can get try it out now.

Reply:
@SondraE @jamesnw 

![image](https://github.com/user-attachments/assets/f1ce6094-e6f2-4763-a9fc-1c62a2b37eb1)

Reply:
lgtm!

Reply:
I realized the section heading was in the wrong column and adjusted that in https://github.com/oddbird/oddleventy/pull/826/commits/ba5768356048413b89ded9f14519bbbb57b0dfc9

![Uploading image.png…]()